pkg/openapi/list.go

```diff
@@ -0,0 +1,72 @@
+package openapi
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/getkin/kin-openapi/openapi3"
+)
+
+type OperationList struct {
+	Operations map[string]Operation `json:"operations"`
+}
+
+type Operation struct {
+	Description string `json:"description,omitempty"`
+	Summary     string `json:"summary,omitempty"`
+}
+
+const NoFilter = "<none>"
+
+func List(t *openapi3.T, filter string) (OperationList, error) {
+	operations := make(map[string]Operation)
+	for _, pathItem := range t.Paths.Map() {
+		for _, operation := range pathItem.Operations() {
+			var (
+				match bool
+				err   error
+			)
+			if filter != "" && filter != NoFilter {
+				if strings.Contains(filter, "*") {
+					var filters []string
+					if strings.Contains(filter, "|") {
+						filters = strings.Split(filter, "|")
+					} else {
+						filters = []string{filter}
+					}
```

Comment:
```suggestion
					filters := strings.Split(filter, "|")
```

If `|` is not in there, Split will just return `[]string{filter}` anyway

Reply:
I feel silly for writing it this way in the first place, lol